### PR TITLE
Cvsl 2071 afer add consumption of domain events to capture when a prisoner is transferred

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

**Comment by nishanthmahasamudram** [comment]
s

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Offender.kt [hunk 1]
```diff
@@ -25,7 +25,7 @@ data class Offender(
   @NotNull
   val prisonerNumber: String,
 
-  val prisonId: String,
+  var prisonId: String,
```
Comment (justin-gardiner) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Offender.kt:28]: Does this need to be var? If you need to update it for an existing offender, could you instead do

val updatedOffender = offender.copy(prisonId = newPrisonId)

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/SampleTest.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi
```
Comment (justin-gardiner) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/SampleTest.kt:1]: Is this still needed?

Comment (nishanthmahasamudram) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/SampleTest.kt:1]: Yes, it's just to fix gradlew test failure.

**Review by justin-gardiner — APPROVED** [review]
LGTM

**Pull request merged**